iothub/device/tests/Authentication/ClientAuthenticationWithTokenRefreshTests.cs

```diff
@@ -240,6 +240,47 @@ public async Task ClientAuthenticationWithSakRefresh_WithModule_SharedAccessKeyC
             Assert.AreEqual(token1, token2.TokenValue);
         }
 
+        [TestMethod]
+        [ExpectedException(typeof(FormatException))]
+        public void ClientAuthenticationWithSakRefresh_InvalidKey_Throws()
+        {
+            var builder = new SharedAccessSignatureBuilder()
+            {
+                Key = TestSharedAccessKey + "123",
```

Comment:
What makes this invalid? May need a code comment.

Reply:
updated